.gitignore

```diff
@@ -20,3 +20,9 @@ __pycache__/
 
 # pytest cache
 tests/.pytest_cache
+
+# venv
+pyvenv.cfg
+/Lib
+/Scripts
+/share
```

Comment:
What files are these? They should probably not be here.

Reply:
I was using venv when testing, they are part of that.